demo/assets/html/panoviewer.html

```diff
@@ -1,28 +1,23 @@
 <div>
-    <style>
-    #panoviewer-showcase {
-        position: relative;
-        width: 100%;
-        height: 400px;
-        margin: 0 auto;
-    }    
-    </style>
-    <div id="panoviewer-showcase">
-
-    </div>
-    <script>
-        var container = document.getElementById("panoviewer-showcase");
-        var PanoViewer = eg.view360.PanoViewer;
-        var panoViewer = new PanoViewer(container, {
-            yaw: 90,
-            image: "./assets/img/test_equi.png"
-        }).on("ready", function() {
-            panoViewer.lookAt({
-                yaw: 0
-            }, 1000);
-        });
-        window.addEventListener("resize", function() {
-            panoViewer.updateViewportDimensions();
-        })
-    </script>
+		<style>
+		#panoviewer-showcase {
+				position: relative;
+				width: 100%;
+				height: 400px;
+				margin: 0 auto;
+		}    
+		</style>
+		<div id="panoviewer-showcase"></div>
+		<script>
+				var container = document.getElementById("panoviewer-showcase");
+				var PanoViewer = eg.view360.PanoViewer;
+				var panoViewer = new PanoViewer(container, {
+					yaw: 180,
+					image: "./assets/img/test_equi.png"
+				}).on("ready", function() {
+					panoViewer.lookAt({
+						yaw: 0
+					}, 2000);
+				});
+		</script>
```

Comment:
This file is for showcase section of the PanoViewer site. So, I think the responsiveness of the viewer should be shown in the showcase.